app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -146,4 +147,10 @@ class LoginFragment : Fragment() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    private fun showSnackbar() {
+        Snackbar.make(
+            rootView.loginCoordinatorLayout, getString(R.string.log_in_first), Snackbar.LENGTH_SHORT
+        ).show()
+    }
```

Comment:
Why is this showed unconditionally?

Reply:
I think there is no need of a check as showing snackbar whenever one need to login is apt. Pls correct me

Reply:
No, if someone goes to login him/herself. There is no need of showing snackbar

Reply:
currently login fragment loads whenever one go for profile or tickets without being logged in and then snackbar is shown